domain/nsenter.go

```diff
@@ -179,3 +181,9 @@ type UmountSyscallPayload struct {
 	FsType uint8  `json:"-"`
 	Flags  uint64 `json:"flags"`
 }
+
+type ChownSyscallPayload struct {
+	Target string `json:"target"`
+	Uid    int    `json:"uid"`
```

Comment:
I find these Uid/Gid fields somewhat confusing as we already have equivalent fields for the process. I'd rather have these call differently, i.e. cUid, cGid.

Reply:
I think the structure sets the context for the fields: in this case it's a chown syscall payload struct, so it's clear the uid/gid fields refer to the chown uid/gid, not the process uid/gid. I could call them targetUid and targetGid if that helps.

Reply:
I'm referring to event.go file. There we are using Uid/Gid fields in a couple of sections while having Uid/Gid carrying different semantics. Also, we may need to include the 'header' struct (where there's already a Uid/Gid) inside this message in the future.

Reply:
I changed it to TargetUid and TargetGid.